core/server/api/canary/utils/serializers/output/preview.js

```diff
@@ -5,5 +5,9 @@ module.exports = {
         frame.response = {
             preview: [mapper.mapPost(model, frame)]
         };
+        const type = model.get('type');
```

Comment:
Should we replace the conditional with something like this?
```js
frame.response.preview[0].page = model.get('type') === 'page';
```

That way the response "shape" is always the same, rather than only including the property for pages?

Reply:
My thought behind that choice was to have as little side effects as possible, so only change the shape of the object if needed.
I agree it would be cleaner to always have the same shape though!